lib/ca.go

```diff
@@ -471,11 +471,21 @@ func (ca *CA) initConfig() (err error) {
 
 // VerifyCertificate verifies that 'cert' was issued by this CA
 // Return nil if successful; otherwise, return an error.
-func (ca *CA) VerifyCertificate(cert *x509.Certificate) error {
+func (ca *CA) VerifyCertificate(cert *x509.Certificate, forceTime bool) error {
```

Comment:
```suggestion
func (ca *CA) VerifyCertificate(cert *x509.Certificate, ignoreCertExpiry bool) error {
```

I think the boolean name should tie back to the config option name.

Reply:
> Need a test for the new config option... unfortunately I don't know enough about how fabric-ca integration test works to provide more specific guidance.

Yes absoltuley.

Reply:
on the name, I'd debated on tieing this to the ignoreCertExpiry but then decided against it
- there are multiple functions that do certificate checks - they are given a cert and then say yes/no;  there is some overlap between the checks they make - expiy being an example
- the controlling function is `verifyX509Token` - by renaming the variable it is pushing knowledge of the config option down into a function that doesn't actually need to know this.
- the controlling function is request a certain style of validation based on what it needs - in this case it needs to have the expiry time be discounted.  

Admit that the multiple checks is a wasteful in processing - and I don't like having to put a defined time to force the expiry check to not be an issue. But changing that is beyond the scope or this pr.

Reply:
ok, let's at least add "forceTime" to the function comment on line 472 and mention that it is used if reenrollIgnoreCertExpiry is set, so that people know when this is used.